phpBB/language/en/common.php

```diff
@@ -397,6 +397,7 @@
 	'LOGIN_CONFIRMATION'				=> 'Confirmation of login',
 	'LOGIN_CONFIRM_EXPLAIN'				=> 'To prevent brute forcing accounts the board requires you to enter a confirmation code after a maximum amount of failed logins. The code is displayed in the image you should see below. If you are visually impaired or cannot otherwise read this code please contact the %sBoard Administrator%s.', // unused
 	'LOGIN_ERROR_ATTEMPTS'				=> 'You exceeded the maximum allowed number of login attempts. In addition to your username and password you now also have to pass the CAPTCHA test.',
+	'LOGIN_ERROR_ATTEMPTS_RECAPTCHA_V3'	=> 'You have exceeded the maximum number of login attempts allowed.<br>In addition to your username and password the invisible reCAPTCHA v3 will be used to authenticate your session.',
```

Comment:
I'm not sure what the proper placement of this comment, but it reads strange for me to say that you should look out for something that you may not even see. Wouldn't it make more sense not to display this message at all with this captcha plugin?

Reply:
There is the reCaptcha logo at the bottom-right side of the screen.

Reply:
Sure. But you might not have to pick out the cars or buses. I'm not even sure when this message would be required at all, as for the other captchas are included in the form. Or at least the ones we ship with core. So I'm not sure if this message really helps/is useful, especially for recaptcha. If you fail the probability test, and pick out some images it would pop up in your face hopefully. If you use another type of captcha, it would be on the form. I'm not sure if this is good for accessability at all, but in my mind that would be the only argument to keep this message at all.

Reply:
We can even suppress it for recaptcha V3 but still there there will be one for recaptcha V2, at this very point I opted for consistency.

Reply:
Yeah, I think this is a good catch, I'm just not sure what the right fix is: changing the message, removing it or maybe something else.

Reply:
Like leaving this as it is could be an option 😉

Reply:
I prefer rephrasing it to not changing it.

Reply:
Okay by my side, no probs. Just give me the new wording.

Reply:
I mean your change to closing the PR. I thought you meant that by leaving it as it is. Sorry for the confusion there.

Reply:
Not sure what I should do?

Reply:
I meant leaving this PR as it is and merge it.